library/src/main/java/com/novoda/downloadmanager/notifications/NotificationTag.java

```diff
@@ -0,0 +1,68 @@
+package com.novoda.downloadmanager.notifications;
+
+import android.app.Notification;
+
+import com.novoda.downloadmanager.lib.DownloadBatch;
+
+/**
+ * Build tag used for collapsing several {@link DownloadBatch} into a single
+ * {@link Notification}.
+ */
+public class NotificationTag {
+
+    private final int status;
+    private final String identifier;
+
+    public static NotificationTag create(DownloadBatch batch, String packageName) {
```

Comment:
We return `null` in the `else`, we could make this super explicit to the clients of this method and add the `@Nullable` annotation.